components/brave_wallet/browser/zcash/rust/Cargo.toml

```diff
@@ -7,7 +7,8 @@ license = "MPL-2.0"
 [dependencies]
 brave_wallet = { version = "1" }
 cxx = { version = "1" }
-orchard = { version = "0.7.1",  default-features = false }
+orchard = { version = "0.8.0",  default-features = false }
```

Comment:
was this bump always here? Maybe I missed it before, but we should not combine the `orchard` crate bump with adding `zcash_primitives` unless it impacts `zcash_primitives` in some important way

Reply:
also need to investigate some things in `orchard` like https://github.com/brave/reviews/issues/1634#issuecomment-2146274646

Reply:
if we do need to combine these for some reason then you need to delete `orchard/v_07` directory

Reply:
zcash_primitives 0.15.1 depends on orchard 0.8 so i've bumped it. We need to use only orchard 0.8
Removed orchard/v_07